clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -179,7 +183,17 @@ export const ConnectorParametersForm = ({
                   size="sm"
                 />
               )}
-              {item.type === "integer" && (
+              {isBoolean && (
+                <AntSelect
```

Comment:
is there a reason we chose to use a select rather than a checkbox which is typically used for booleans on a form?

Reply:
I used a checkbox initially, which I think it's better UX. 
But with the current layout for the form, it looks weird because of the empty space it creates and having the ? icon all the way to the right. Jack asked me to change that to a Select instead.
<img width="981" alt="Captura de pantalla 2024-12-03 a la(s) 1 09 50 p  m" src="https://github.com/user-attachments/assets/b13b7903-b5ec-4a6c-b0b1-4098e3077927">

I think this is still an improvement over the input text, but in the future I would like to see a layout that supported the checkbox better.